tests/foreman/installer/test_installer.py

```diff
@@ -1390,6 +1393,45 @@ def test_installer_options_and_sections(filter):
     assert previous == current, msg
 
 
+@pytest.mark.e2e
+@pytest.mark.tier1
+@pytest.mark.parametrize("sat_ready_rhel", [settings.server.version.rhel_version], indirect=True)
+def test_satellite_installation(sat_ready_rhel):
+    """Run a basic Satellite installation
+
+    :id: bbab30a6-6861-494f-96dd-23b883c2c906
+
+    :steps:
+        1. Get a RHEL host
+        2. Configure repos
+        3. Enable satellite module
+        4. Install satellite
+        5. Run satellite-installer
+
+    :expectedresults:
+        1. Correct satellite packaged is installed
+        2. satellite-installer runs successfully
+        3. Health check runs successfully
+
+    :CaseImportance: High
+    """
+    sat_version = settings.server.version.release
+    # Register for RHEL8 repos, get Ohsnap repofile, and enable and download satellite
+    sat_ready_rhel.register_to_cdn()
+    sat_ready_rhel.download_repofile(product='satellite', release=settings.server.version.release)
+    sat_ready_rhel.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
+    installed_version = sat_ready_rhel.execute('rpm -qa satellite').stdout
+    assert sat_version in installed_version
+    # Install Satellite
+    result = sat_ready_rhel.execute(
+        InstallerCommand('scenario satellite').get_command(), timeout='30m'
+    )
```

Comment:
non-blocking, can we also extend this test for scenario capsule and either parametrize this or have separate test for capsule install on sat_ready_rhel, wdyt?

Reply:
I would say that's up to Team Endeavour. I would expect that to be a separate test (if it doesn't already exist), as this is more of just a quick e2e test for installer on stream.

Reply:
@pondrejk thoughts?

Reply:
I think the notion of does a Capsule properly install is on our team, since that is controlled by packaging + installer scenario and other architectural aspects. And one that we should vet as an e2e to ensure we do not break anything.

Reply:
Is it safe to say that should be a separate test? I can open another PR for that

Reply:
Yes, it can be made a separate test or can be parameterized as well

Reply:
> Is it safe to say that should be a separate test? I can open another PR for that

Ideally, probably yes. But I think it's fair to weigh the cost. Since you have to have a Satellite already configured and installed to test a Capsule install, doing it as a follow on when you've already got a basic Satellite installed is fair to save time and resources.